tests/kernel/workq/work_queue/src/main.c

```diff
@@ -722,6 +722,41 @@ void test_delayed_work_define(void)
 			  sizeof(struct k_delayed_work), NULL);
 }
 
+/**
+ * @brief Verify k_work_poll_cancel()
+ *
+ * @ingroup kernel_workqueue_tests
+ *
+ * @details Cancel a triggered work item repeatedly,
+ * see if it returns expected value.
+ *
+ * @see k_work_poll_cancel()
+ */
+static void test_triggered_cancel(void)
+{
+	int ret;
+
+	TC_PRINT("Starting triggered test\n");
+
+	/* As work items are triggered, they should indicate an event. */
+	expected_poll_result = 0;
+
+	TC_PRINT(" - Initializing triggered test items\n");
+	test_triggered_init();
+
+	TC_PRINT(" - Submitting triggered test items\n");
```

Comment:
I think we don't need this TC_PRINT here because there is already one inside the test_triggered_submit() you called. See line 434.